cli/src/command/wallet.rs

```diff
@@ -107,6 +109,20 @@ impl Default for InitParameters {
     }
 }
 
+pub async fn accounts_command(storage_path: &Path, snapshot_path: &Path) -> Result<(), Error> {
+    let password = get_password("Stronghold password", false)?;
+    let wallet = unlock_wallet(storage_path, snapshot_path, password).await?;
+    let accounts = wallet.get_accounts().await?;
+
+    println!("INDEX\tALIAS");
```

Comment:
Why scream

Reply:
I tried both ... and having it all small letters doesn't make it distinguishable enough for eye to separate it from actual aliases, like:
```
index    alias
0        alice
```
compared to
```
INDEX    ALIAS
0        alice
```
which I find easier to parse.

Reply:
fair enough

Reply:
obviously that would be my preference, but I don't case so much tbh.